playground/AzureFunctionsEndToEnd/AzureFunctionsEndToEnd.AppHost/funcstorage67c6c.module.bicep

```diff
@@ -0,0 +1,77 @@
+@description('The location for the resource(s) to be deployed.')
+param location string = resourceGroup().location
+
+param principalId string
+
+param principalType string
+
+resource funcstorage67c6c 'Microsoft.Storage/storageAccounts@2024-01-01' = {
+  name: take('funcstorage67c6c${uniqueString(resourceGroup().id)}', 24)
```

Comment:
`funcstorage67c6c`  is 16 chars, so we will have 8 chars of uniquestring. I _think_ we should be OK here.